vm/src/obj/objnone.rs

```diff
@@ -111,6 +111,24 @@ impl PyNoneRef {
             Err(vm.new_attribute_error(format!("{} has no attribute '{}'", self.as_object(), name)))
         }
     }
+
+    #[pymethod(name = "__eq__")]
+    fn eq(self, rhs: PyObjectRef, vm: &VirtualMachine) -> PyResult {
```

Comment:
When returning value doesn't have `Err` case, return type simply can be `PyObjectRef` instead of `PyResult`